crates/runtime/src/actor/component/mod.rs

```diff
@@ -445,7 +445,9 @@ fn instantiate(
     let mut custom_result_types = HashMap::with_capacity(imports.len());
     for (instance_name, item) in imports {
         match instance_name {
-            "wasi:config/runtime@0.2.0-draft"
+            "wasi:blobstore/blobstore@0.2.0-draft"
+            | "wasi:blobstore/container@0.2.0-draft"
+            | "wasi:blobstore/types@0.2.0-draft"
```

Comment:
It could be worth adding a comment here to indicate what interfaces should be ignored here